buildSrc/src/main/resources/forbidden/es-test-signatures.txt

```diff
@@ -25,3 +25,8 @@ org.apache.lucene.util.LuceneTestCase$Nightly @ We don't run nightly tests at th
 com.carrotsearch.randomizedtesting.annotations.Nightly @ We don't run nightly tests at this point!
 
 org.junit.Test @defaultMessage Just name your test method testFooBar
+
+@defaultMessage Explicitly specify the ContentType of HTTP entities
+org.apache.http.entity.StringEntity#<init>(java.lang.String)
+org.apache.http.entity.StringEntity#<init>(java.lang.String,java.lang.String)
+org.apache.http.entity.StringEntity#<init>(java.lang.String,java.nio.charset.Charset)
```

Comment:
++ I should add all of those here. The difference is the http-signatures file is used only for the rest projects as they do not use the es-test-signatures. 

I could move the http signatures to the buildSrc and then have the default forbidden apis for tests include that file as well. This way we only need to update the file in one place.